dbhydra/dbhydra_core.py

```diff
@@ -723,6 +761,76 @@ def create(self, foreign_keys=None):
             print("Table " + self.name + " already exists:", e)
             print("Check the specification of table columns and their types")
 
+
+    def insert(self,rows,batch=1,replace_apostrophes=True,try_mode=False, debug_mode=False):
```

Comment:
this is scary, how much would we improve code if we used execute with params? this is to reference to above comment as well. 
I wonder how these formatting issues are tackled with parameter execute, whether it would take less than 20 rows